src/pwncore/models/user.py

```diff
@@ -37,6 +34,7 @@ async def save(self, *args, **kwargs):
 
 
 class Team(Model):
+    id = fields.IntField(pk=True)  # team.id raises Team does not have id, so explicitly adding it
```

Comment:
That's weird, what's the code that was doing that?

Reply:
Not sure how to reference a line in a comment, I'll just copy paste the code here:

Line 63 of src/pwncore/routes/auth.py:
```py
@router.post("/login")
async def team_login(team_data: LoginBody, response: Response):
    # TODO: Simplified logic since we're not doing refresh tokens.

    team = await Team.get_or_none(name=team_data.name)
    if team is None:
        response.status_code = 404
        return {"msg_code": config.msg_codes["team_not_found"]}
    if not bcrypt.verify(team_data.password, team.secret_hash):
        response.status_code = 401
        return {"msg_code": config.msg_codes["wrong_password"]}

    current_time = datetime.datetime.utcnow()
    expiration_time = current_time + datetime.timedelta(hours=config.jwt_valid_duration)
 >   token_payload = {"team_id": team.id, "exp": expiration_time}
    token = jwt.encode(token_payload, config.jwt_secret, algorithm="HS256")

    # Returning token to be sent as an authorization header "Bearer <TOKEN>"
    return {
        "msg_code": config.msg_codes["login_success"],
        "access_token": token,
        "token_type": "bearer",
    }
```